draft-ietf-mls-architecture.md

```diff
@@ -205,18 +205,11 @@ client components; and they may even involve some action by users.  For example:
 
 It is important to note that the Authentication Service functionality
 can be completely abstract in the case of a Service Provider which
-allows MLS clients to generate, redistribute and validate their
+allows MLS clients to generate, distribute, and validate
 credentials themselves.
-
-Similarly to the AS, the Delivery Service can be completely abstract
+Similar to the AS, the Delivery Service can be completely abstract
 if users are able to distribute credentials and messages without
-relying on a central Delivery Service. Note, though, that the MLS
-protocol requires group operation messages to be processed in-order by
-all MLS clients.
```

Comment:
Removed because strict in-order processing isn't required

Reply:
Is that true? They don't need to be received in order but they do need to be processed in the order of their epoch no? CC @bifurcation

Reply:
This is discussed more in #113 but the state of different clients can fork based on the consistency model of the DS.

Reply:
Seems like we can just generalize here, something like:

> Note, though, that in such scenarios, clients will need to implement logic that assures the delivery properties required of the DS (see {{delivery-guarantees}})